src/context/UserAccessContext/permissions.ts

```diff
@@ -0,0 +1,42 @@
+export const tables = {
+  database: "database",
+  rbac: "rbac",
+  auth: "auth",
+  responder: "responder",
+  incident: "incident",
+  evidences: "evidences",
+  comments: "comments",
+  canaries: "canaries",
+  topologies: "topologies",
+  config_scrapers: "config_scrapers",
+  identities: "identities",
+  connections: "connections",
+  kratos: "kratos"
+};
+
+export const permDefs = {
+  admin: {
+    read: [...Object.values(tables)],
+    write: [...Object.values(tables)]
+  },
+  editor: {
+    read: [tables.canaries, tables.topologies, tables.config_scrapers],
```

Comment:
editor can read everything but connections
and write to everything except connections and users

commander and responder both extend the viewer role
responders have the same table permissions as commanders, but responders cannot update the hypothesis status or change the incident status or commander - these lower level permissions we can however defer

Please also test all the roles - There shouldn't. be any errors shown when navigating